apollo-runtime/src/commonMain/kotlin/com/apollographql/apollo3/network/ws/WebSocketNetworkTransport.kt

```diff
@@ -166,6 +165,12 @@ private constructor(
             return
           }
 
+          if (message is Close) {
+            messages.trySend(NetworkError(message.reason))
+            closeProtocol()
```

Comment:
Do you need to call `closeProtocol()` here given that it'll be closed when handling the `NetworkError` message.

2nd question: would it be worth reusing the `NetworkError` and save a queue/dequeue? Not that it is that important but it feels like the semantics are the same?

Reply:
Good point! `NetworkError` was enough, no need for specific command 👍